### PR TITLE
**Feature:** Enhance Tree component

### PR DESCRIPTION
**Pull request by fabien0102**
# Summary

Tweak the design and permit to have icons! 

image

# Related issue

https://contiamo.atlassian.net/browse/UI-47

# To be tested

Me
- [x] No error or warning in the console on `localhost:6060`

Tester 1

- [x] Things look good on the demo.
- [x] I can add an icon to a child
- [x] I can have a custom icon color
- [x] I can put two letters in a tag

Tester 2

- [ ] Things look good on the demo.
- [ ] I can add an icon to a child
- [ ] I can have a custom icon color
- [ ] I can put two letters in a tag

### CODE REVIEW

File: src/NameTag/NameTag.tsx [hunk 1]
```diff
@@ -41,12 +41,14 @@ const Container = styled("div", {
...
   const textColor = readableTextColor(backgroundColor, [theme.color.white, theme.color.black])
+
   return {
     backgroundColor,
     color: textColor,
-    fontSize: condensed ? theme.font.size.tiny : theme.font.size.small,
+    fontSize: condensed ? theme.font.size.tag : theme.font.size.small,
     fontWeight: theme.font.weight.bold,
-    width: condensed ? 14 : 28,
+    fontStretch: "condensed",
```
Comment (TejasQ) [inline src/NameTag/NameTag.tsx]: wow whats this

Comment (TejasQ) [reply src/NameTag/NameTag.tsx]: image

**Comment by stereobooster** [comment]
Screenshot 2019-06-12 at 10 21 05

Icon doesn't have hand cursor and it needs some spacing on the left in hover state (maybe?)

**Comment by stereobooster** [comment]
Can we use font weight normal for highlight, because otherwise it stretches labels when you type?

**Comment by fabien0102** [comment]
> Can we use font weight normal for highlight, because otherwise it stretches labels when you type?

Just following the design for this, let's merge this, we can still iterate on this later ;)

**Pull request merged**